### PR TITLE
[ui][new] API Provider tests

### PR DESCRIPTION
**Pull request by asmigala**
//test: @api-provider

### CODE REVIEW

File: utilities/src/main/java/io/syndesis/qe/utils/RestUtils.java [hunk 1]
```diff
@@ -103,7 +108,8 @@ private static HttpClient createAllTrustingClient() throws RestClientException {
             final SSLContextBuilder builder = new SSLContextBuilder();
             builder.loadTrustMaterial((TrustStrategy) (X509Certificate[] chain, String authType) -> true);
             final SSLConnectionSocketFactory sslsf = new SSLConnectionSocketFactory(
-                    builder.build());
+                    builder.build(),
+                    new NoopHostnameVerifier()); // needed to connections to API Provider integrations
```
Comment (asmigala) [inline utilities/src/main/java/io/syndesis/qe/utils/RestUtils.java]: @tplevko not sure this is ok, should I rewrite it to have a specific client for api provider?

**Comment by syndesisqe** [comment]
Build finished. Tests PASSED

**Comment by tplevko** [comment]
retest this please

File: utilities/src/main/java/io/syndesis/qe/utils/RestUtils.java (hunk 1 above)
Comment (tplevko) [reply utilities/src/main/java/io/syndesis/qe/utils/RestUtils.java]: @asmigala I believe this won't be an issue, as this client is solely used for testing purposes

**Comment by syndesisqe** [comment]
Build finished. Tests FAILED

**Comment by tplevko** [comment]
retest this please

**Comment by syndesisqe** [comment]
Build finished. Tests FAILED

**Comment by syndesisqe** [comment]
Build finished. Tests FAILED

**Comment by tplevko** [comment]
retest this please

**Comment by syndesisqe** [comment]
Build finished. Tests FAILED

**Pull request merged**